lib/lita/adapters/slack/message_handler.rb

```diff
@@ -41,7 +41,9 @@ def handle
 
         def body
           normalized_message = if data["text"]
-            data["text"].sub(/^\s*<@#{robot_id}>/, "@#{robot.mention_name}")
+            data["text"].sub(/^\s*<@#{robot_id}>/, "@#{robot.mention_name}") #original
+            # data["text"].sub(/^/[[:space:]]/*<@#{robot_id}>/, "@#{robot.mention_name}") #never works
+            # data["text"].to_s.gsub(/[[:space:]]/,' ').sub(/^\s*<@#{robot_id}>/, "@#{robot.mention_name}") #works but not ideal
```

Comment:
Instead of leaving commented-out code in the file, I'd suggest moving the various options into the git commit message, and use full paragraphs to describe the "why" of the option you chose